### DISCUSSION TITLE
Combobox Selected Label

### DISCUSSION THREAD

**Discussion by noynek** [microsoft/fluentui#26092] [Q&A] [answered]
Hey, I'm new to Fluent UI, so hopefully, I'm just missing something. I'm building a "State" combobox with options like:
`[
 {
   label: "Arizona",
  value: "AZ"
 } 
]`

Most things work, but I can't figure out how to make the selected value the `label`. It currently displays the `value`. 
<img width="363" alt="image" src="https://user-images.githubusercontent.com/2458230/209261028-c4fde4e1-a576-458d-bf82-c49b78bd5515.png">

```
<Combobox
   selectedOptions={value.selected ? value.selected : ['']}
   aria-labelledby={comboId}
   onChange={onChangeHandler}
   onOptionSelect={onOptionSelectHandler}
  >
    {matchingOptions.map((option: { value: string; label: string }) => (
     <Option key={option.value} value={option.value}>
      {option.label}
       </Option>
       ))}
</Combobox>
```

**Comment by micahgodbolt** [comment, marked as answer]
This is being addressed here: https://github.com/microsoft/fluentui/pull/26617

Happy to take feedback if you have any.
